pkg/lib/bundle/build.go

```diff
@@ -4,22 +4,19 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"path"
 
 	log "github.com/sirupsen/logrus"
 )
 
 // Create build command to build bundle manifests image
-func BuildBundleImage(directory, imageTag, imageBuilder string) (*exec.Cmd, error) {
+func BuildBundleImage(imageTag, imageBuilder string) (*exec.Cmd, error) {
 	var args []string
 
-	dockerfilePath := path.Join(directory, DockerFile)
-
 	switch imageBuilder {
 	case "docker", "podman":
-		args = append(args, "build", "-f", dockerfilePath, "-t", imageTag, directory)
+		args = append(args, "build", "-f", DockerFile, "-t", imageTag, ".")
```

Comment:
This needs to be tested out because I'm pretty sure that this won't work if the opm binary is not in the same directory with the manifests. I recall that I changed this from `.` to `diretory` to avoid this issue.